clio-server/src/main/scala/org/broadinstitute/clio/server/service/IndexService.scala

```diff
@@ -147,6 +148,7 @@ abstract class IndexService[CI <: ClioIndex](
     existingMetadata: clioIndex.MetadataType,
     newMetadata: clioIndex.MetadataType
   ): Either[Exception, clioIndex.MetadataType] = {
+    implicit val foo = new Patience[Json]
```

Comment:
Debugging line?

Reply:
I think that this is something which tells the test to wait for the completion of the task which returns the future. Dan suggested that it should be added and I'm not sure whether we can delete it or not. I'll test, and comment with the answer.

Reply:
It requires the Patience to be added, if not then the diff fails as this:
[error] /Users/ranthony/GitHub/Broad/clio/clio-server/src/main/scala/org/broadinstitute/clio/server/service/IndexService.scala:152: could not find implicit value for parameter Diff: diffson.Diff[io.circe.Json,P]
[error]     val differences = diff(existingMetadata.asJson, newMetadata.asJson)
[error]                           ^
[error] /Users/ranthony/GitHub/Broad/clio/clio-server/src/main/scala/org/broadinstitute/clio/server/service/IndexService.scala:9: Unused import
[error] import diffson.lcs.Patience
[error]                    ^
[error] /Users/ranthony/GitHub/Broad/clio/clio-server/src/main/scala/org/broadinstitute/clio/server/service/IndexService.scala:152: local val differences in method validateUpsert is never used
[error]     val differences = diff(existingMetadata.asJson, newMetadata.asJson)
[error]         ^
[error] /Users/ranthony/GitHub/Broad/clio/clio-server/src/main/scala/org/broadinstitute/clio/server/service/IndexService.scala:148: parameter value existingMetadata in method validateUpsert is never used
[error]     existingMetadata: clioIndex.MetadataType,
[error]     ^

Reply:
Looks like an implicit argument to the following line `val differences = diff(…)`
to specify a "longest common subsequence" algorithm optimized for JSON
for use with the Circe library. I don't know why, but can demonstrate how to
get IntelliJ to show you what I mean.

That implicit was probably provided by one of the other imports before the
library update, but now we have to do it to avoid the `Unused import` error.